pipa.json

```diff
@@ -0,0 +1,28 @@
+{
+    "response": [
+        {
+            "maintainer": Varun Soma"",
+            "oem": "Xiaomi",
+            "device": "Pad 6",
+            "filename": "crDroidAndroid-15.0-20250210-pipa-v11.2.zip",
+            "download": "https://sourceforge.net/projects/crdroid/files/pipa/11.x/crDroidAndroid-15.0-20250210-pipa-v11.2.zip/download",
+            "timestamp": 1739194268,
+            "md5": "b2f1f618701ee835974eb74f7847bdd4",
+            "sha256": "9c2488fa924809f344a1d7ee805d37ba4bfb7f2f791c023b2bd7c902bd89f75f",
+            "size": 1320546078,
+            "version": "11.2",
+            "buildtype": "Monthly",
+            "forum": "https://xdaforums.com/t/beta-crdroid-11-x-a15-pipa.4641070/",
```

Comment:
update forum to new template
https://github.com/crdroidandroid/crdroid_features/blob/15.0/ForumPostTemplate